src/index.js

```diff
@@ -0,0 +1,147 @@
+// --------------------------------wave3 and 6 Naming the City----------------------
+const cityNameLabel = document.getElementById('cityname');
+const cityNameInput = document.getElementById('cityname-input');
+const resetBtn = document.getElementById('reset-btn');
+// change city name on top header
+const addCityName = document.getElementById('topCityName');
+var currCity;
+// handle Get Realtime Temperature button to check weather
+const gettemperature = document.getElementById('gettemperature');
+var count;
```

Comment:
These should be moved to a callback that fires once the DOM content is loaded. Otherwise there's a risk that the elements may not have been created yet.